velox/functions/sparksql/registration/RegisterJson.cpp

```diff
@@ -24,6 +26,8 @@ void registerJsonFunctions(const std::string& prefix) {
       {prefix + "get_json_object"});
   registerFunction<JsonObjectKeysFunction, Array<Varchar>, Varchar>(
       {prefix + "json_object_keys"});
+  registerFunction<JsonArrayLengthFunction, int32_t, Varchar>(
+      {prefix + "json_array_length"});
```

Comment:
Please also document this function in https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/json.rst. Thanks.

Reply:
done